app.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+$('.checkbox').on('click', function () {
+  $(event.target).toggleClass('unchecked, checked');
+});
+
+$('.radio').on('click', function () {
```

Comment:
I'd set this up, not just to toggle the class on `event.target`, but to toggle it on all of the `.radio` elements. That way it correctly shows/unshows both of the radio buttons as being selected.